KCs-Linux-Scripts/Linux-Auto-Compile-Best-Of-Both-Worlds.sh

```diff
@@ -1,13 +1,14 @@
 #/bin/bash
 
-cd /home/$(whoami)/
+cd ~
 
 git clone https://github.com/wabbajack-tools/ussedp
 
 cd ussedp
 
-read -s -p "Enter Password for sudo: " sudoPW
-echo $sudoPW | sudo -S dotnet build
```

Comment:
this is actually important because when the downloader gets too working it could take more than the 10 min given by sudo and not all distros are 10 min some are 10 sec default

Reply:
`NOT` a problem
my new script only has **this one sudo**